tests/test-cases/Group8-vSphere-Integration/8-02-OOB-VM-Register.robot

```diff
@@ -19,21 +19,17 @@ Suite Teardown  Extra Cleanup
 
 *** Keywords ***
 Extra Cleanup
-    ${out}=  Run Keyword And Ignore Error  Run  govc vm.destroy ${old-vm}
-    ${out}=  Run Keyword And Ignore Error  Run  govc pool.destroy host/*/Resources/${old-vm}
-    ${out}=  Run Keyword And Ignore Error  Run  govc datastore.rm ${old-vm}
-    ${out}=  Run Keyword And Ignore Error  Run  govc host.portgroup.remove ${old-vm}-bridge
-    Cleanup VIC Appliance On Test Server
+    Run Keyword And Continue On Failure  Manually Cleanup VCH  %{GROUP-8-VCH}
+    Run Keyword And Continue On Failure  Cleanup VIC Appliance On Test Server
 
 *** Test Cases ***
 Verify VIC Still Works When Different VM Is Registered
-    Install VIC Appliance To Test Server
+    Install VIC Appliance To Test Server  cleanup=${false}
     Set Suite Variable  ${old-vm}  %{VCH-NAME}
-    Add VCH to Removal Exception List  ${old-vm}
+    Set Environment Variable  GROUP-8-VCH  ${old-vm}
```

Comment:
Did we mean `Set Suite Variable`?

Reply:
Nope, in this case we want this to be an environment variable. This test will get more reworking in the future as it has always leaked bridge networks. So it is not worth the time to tweak it further.